ConfigVersionControl/config_version_control.py

```diff
@@ -11,9 +15,20 @@ def __init__(self, working_directory):
     #i.e. not automatically via the file observer firing an event, but after a new configuration is created
```

Comment:
Looking closer: what are these comments related to? If it is a method then they should be inside it